project/plugins.sbt

```diff
@@ -1,11 +1,13 @@
 resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("com.typesafe.play"    %  "sbt-plugin"             % "2.8.18")
-addSbtPlugin("uk.gov.hmrc"          %  "sbt-auto-build"         % "3.9.0")
+addSbtPlugin("com.typesafe.play"    %  "sbt-plugin"             % "2.8.19")
+addSbtPlugin("uk.gov.hmrc"          %  "sbt-auto-build"         % "3.13.0")
```

Comment:
3.18.0 should work and is the latest

Reply:
Already done

Reply:
My mistake - was looking at an older commit